kano_settings/system/advanced.py

```diff
@@ -490,6 +489,31 @@ def set_dns_parental(enabled):
     refresh_resolvconf()
 
 
+def set_youtube_cookies(enabled):
+    # The cookie enables/disables safety mode in YouTube (Midori)
+    # The .db files are located in /usr/share/kano-video
+    # It needs to be copied to /home/USERNAME/.config/midori (browser) and
+    # /home/USERNAME/.config/midori/youtube (kano-video-browser)
+
+    midori_cookie_path = '/home/{}/{}'.format(username, midori_cookie)
+    youtube_cookie_path = '/home/{}/{}'.format(username, youtube_cookie)
```

Comment:
I think that the parental mode is set for the whole system, not per user (as the [lock file] is `/etc/kano-parental-lock` and the hosts file is system-wide). This means that we'll need to copy the Midori and Youtube cookies to each home dir (unless there is a system-wide config for Midori in `/etc`?)